File: party.py
import requests
import datetime
import json

SITE = 'https://api.meetup.com/2/open_events'
API = '25761a5a345a4f53585f6e5427c5917'


DATE = datetime.datetime.now().toordinal() * 24 * 60 * 60
week = 24 * 7 * 60 * 60

CATEGORY = 34
COUNTRY = 'US'
STATE = 'NY'
CITY = 'Brooklyn'

params = {
    'key': API,
    'country': COUNTRY,
    'state': STATE,
    'radius': 1,
    'city': CITY,
    'order': 'time',
    'category': CATEGORY,
    'time': str(DATE) + ", " + str(DATE) + str(week)}

r = requests.get(SITE, params=params)
j = json.loads(r.text)

f = open('html.html', 'w', encoding='utf-8')
if j['results']:
    f.write('Url: ' + str(r.url) + '<hr/>')
    for i in j['results']:
        if i.get('venue')  and i.get('description'):
            message = '<h1>' + str(i['name']) + '</h1>' + \
                      '<h3>' + ' Adress: ' + i['venue']['city'] + ', ' + i['venue']['address_1'] + '</h3>' + '\n' + i[
                          'description'] + '\n' + '<h3>' + str(
                datetime.datetime.fromtimestamp(int(i['time']) / 1e3)) + '</h3>' + '\n' + '------------\n'
            f.write(message)
    print('lucky')
else:
    f.write('empty')
    print('empty')
f.close()

File: party.js
var url = 'https://api.meetup.com/2/open_events',
    api = '25761a5a345a4f53585f6e5427c5917';

var d = new Date();
var dat = d.getTime();
var week = 24 * 7 * 60 * 60;
var enddat = dat+week;
var category = 34,
    country = 'US',
    state = 'NY',
    city = 'Brooklyn',
    radius = 2,
    time = ''+dat + ", "+enddat;

var fs = require('fs');
var htmlcompl = '';

var request = require('request');
request({
    url: url,
    qs: {'key': api, 'country': country, 'category': category, 'state': state, 'city ': city, 'radius': radius},
    method: 'GET'

}, function(error, response, body){
    if(error) {
        console.log(error);
    } else {
        var response = JSON.parse(body);
        if (response['results'].length > 0){
            for (index = 0; index < response['results'].length; ++index) {
                if ('venue' in response['results'][index] && 'description' in response['results'][index]){
                    var tim = new Date(response['results'][index]['time']);

                    htmlcompl += '<h1>' + response['results'][index]['name']+ '</h1>' + ' \n' +
                        '<h3>' + ' Adress: ' + response['results'][index]['venue']['city'] +
                        ', ' + response['results'][index]['venue']['address_1'] + '</h3>' +'</h3>' + '\n' +
                        response['results'][index]['description']+ '\n' + '<h3>' + tim.toString() + '</h3>' +
                        '\n' + '------------\n';
                }
            }
            fs.writeFileSync('test.html', htmlcompl);
        }

    }

});
